dl/src/main/scala/com/intel/analytics/bigdl/nn/Reshape.scala

```diff
@@ -36,8 +51,8 @@ class Reshape[@specialized(Float, Double) T: ClassTag](
 
   override def updateOutput(input: Tensor[T]): Tensor[T] = {
 
-    if ((batchMode.nonEmpty && batchMode.get == false) ||
-      (input.nElement() == nElement && batchMode.isEmpty && input.size(1) != 1)) {
+    if (batchMode.nonEmpty && !batchMode.get ||
```

Comment:
There is a yellow warning, then Intellij remove it automatically. If you think the parentheses make the line more readable, I will add them back :-)